### PR TITLE
Remove MapStore for flux v3.0

### PR DESCRIPTION
**Pull request by tht13**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/facebook/flux/blob/master/CHANGELOG.md
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

**Comment by andy-ms** [comment]
Change the first line of `flux/index.d.ts` to `// Type definitions for Flux 3.0`.
Also, will the other changes mentioned in the changelog affect typings?

**Comment by tht13** [comment]
@andy-ms As far as I can tell the new commit has all the new changes to exposed api methods, please review again

**Comment by andy-ms** [comment]
I'll leave 3 days for authors to review. If they haven't responded in 3 days, ping me.

**Comment by stkb** [comment]
Sorry can't review as I'm not using this any more.

File: flux/package.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
```
Comment (andy-ms) [inline flux/package.json]: If there are no dependencies you can remove this whole file.

**Comment by GiedriusGrabauskas** [comment]
Thanks 👍

**Pull request merged**